src/fluree/db/conn/s3.clj

```diff
@@ -17,14 +17,14 @@
             [fluree.db.util.core :as util]
```

Comment:
Not for this PR, but this ns seems barely S3-specific now. Is there an opportunity for more code consolidation emerging here?